Sources_v3/StreamChat/Models/Attachment.swift

```diff
@@ -168,6 +168,9 @@ public struct AttachmentAction: Codable, Hashable {
         self.type = type
         self.text = text
     }
+
+    /// Check if the action is cancel button.
+    public var isCancel: Bool { value == "cancel" }
```

Comment:
I know this is not our fault but "MY EYES! MY EYES!!!" 😱 . We should probably make sure it works well for `Cancel` or `CANCEL`

Reply:
made `case insensitive` and added test in 95ef457bd3a7dcc4a17db0ba7be78b7bb16c9956